src/CameraControls.ts

```diff
@@ -616,6 +587,51 @@ export class CameraControls extends EventDispatcher {
 
 	}
 
+	get camera() {
+		return this._camera;
+	}
+
+	set camera(camera: _THREE.PerspectiveCamera | _THREE.OrthographicCamera) {
+		this._camera = camera;
+
+		// camera up
+		this._yAxisUpSpace = new THREE.Quaternion().setFromUnitVectors( this._camera.up, _AXIS_Y );
+		this._yAxisUpSpaceInverse = quatInvertCompat( this._yAxisUpSpace.clone() );
+
+		// rotation
+		this._spherical = new THREE.Spherical().setFromVector3( _v3A.copy( this._camera.position ).applyQuaternion( this._yAxisUpSpace ) );
+		this._sphericalEnd = this._spherical.clone();
```

Comment:
I think `this._yAxisUpSpace`, `this._spherical`, and other settings should be shared with cameras, weather than making new instances.

Reply:
@Fennec-hub
I think `set camera` should only change `this._camera` and the CameraControls instance should keep the same state, such as `this._yAxisUpSpace`, `this._spherical`, and other props.

otherwise, `set camera` could change props accidentally, I guess...